.travis.yml

```diff
@@ -1,18 +1,41 @@
 dist: xenial
 language: python
-python:
-  - "2.7"
-  - "3.5"
-  - "3.6"
-  - "3.7"
+matrix:
+  include:
+    - name: Ansible devel with Python 2.7
```

Comment:
This may sound a little silly, but you could either swap the order of the variables or the wording to they match? Makes trying to see the matrix easier for me.

Reply:
![matrix](https://pixel.nymag.com/imgs/daily/vulture/2019/02/07/07-matrix-1.w570.h712.jpg)